frontend/src/components/Forms/TeamEventCreditsForm/TeamEventsCreditDashboard.tsx

```diff
@@ -53,6 +57,48 @@ const TeamEventCreditDashboard = (props: {
         : ''
     }.`;
 
+  const getHoursAttended = (attendance: TeamEventAttendance): number => {
+    const hours = attendance.hoursAttended;
+    if (hours !== undefined) return hours;
+    return 1;
+  };
+
+  const TecDetailsDisplay = (props: { attendanceList: TeamEventAttendance[] }): JSX.Element => {
+    const { attendanceList } = props;
+    return (
+      <Card.Group>
+        {attendanceList.map((attendance) => {
+          const teamEvent = allTEC.find((tec) => tec.uuid === attendance.eventUuid);
+          if (teamEvent !== undefined) {
+            return (
+              <Card>
+                <Card.Content>
+                  <Card.Header>{teamEvent.name} </Card.Header>
+                  <Card.Meta>{teamEvent.date}</Card.Meta>
+                  <Card.Meta>
+                    {`Number of Credits: ${
+                      teamEvent.hasHours
+                        ? getHoursAttended(attendance) * Number(teamEvent.numCredits)
+                        : teamEvent.numCredits
+                    }`}
+                  </Card.Meta>
+                  {COMMUNITY_EVENTS && (
+                    <Card.Meta>Community Event: {teamEvent.isCommunity ? 'Yes' : 'No'}</Card.Meta>
+                  )}
+                </Card.Content>
+              </Card>
+            );
+          }
+          return (
+            <Message>
+              The team event for attendance {attendance.uuid} cannot be found. Contact leads.
```

Comment:
This seems like more of an #idol-support question, not a leads question.

Reply:
Sounds good!